cedar-wasm/build-wasm.sh

```diff
@@ -62,21 +67,21 @@ fix_package_json_files() {
 process_types_file() {
     local types_file="$1"
     echo "processing types file: $1"
-    sed -i "s/[{]\s*!: /{ \"!\": /g" "$types_file"
-    sed -i "s/[{]\s*==: /{ \"==\": /g" "$types_file"
-    sed -i "s/[{]\s*!=: /{ \"!=\": /g" "$types_file"
-    sed -i "s/[{]\s*<: /{ \"<\": /g" "$types_file"
-    sed -i "s/[{]\s*<=: /{ \"<=\": /g" "$types_file"
-    sed -i "s/[{]\s*>: /{ \">\": /g" "$types_file"
-    sed -i "s/[{]\s*>=: /{ \">=\": /g" "$types_file"
-    sed -i "s/[{]\s*&&: /{ \"\&\&\": /g" "$types_file"
-    sed -i "s/[{]\s*||: /{ \"||\": /g" "$types_file"
-    sed -i "s/[{]\s*[+]: /{ \"+\": /g" "$types_file"
-    sed -i "s/[{]\s*-: /{ \"-\": /g" "$types_file"
-    sed -i "s/[{]\s*[*]: /{ \"*\": /g" "$types_file"
-    sed -i "s/[{]\s*\.: /{ \".\": /g" "$types_file"
-    sed -i "s/ | __skip//g" "$types_file"
-    sed -i "s/ { .\+: __skip } |//g" "$types_file"
+    sed -i '' "s/[{]\s*!: /{ \"!\": /g" "$types_file"
```

Comment:
For mac/linux portability, IIRC, I've had to write `-i'.bak'` (no space after `-i`). Then you'll want to include a `rm `"$types_file".bak` to clean up.

Reply:
also, we  could do this with one `sed` command:

```bash
sed -i'.bak' "
s/[{]\s*!: /{ \"!\": /g;
s/[{]\s*==: /{ \"==\": /g;
...
" "$types_file"
```

Reply:
Thanks for the tip. Done.